esp-backtrace/src/lib.rs

```diff
@@ -1,6 +1,25 @@
 #![allow(rustdoc::bare_urls, unused_macros)]
 #![cfg_attr(target_arch = "xtensa", feature(asm_experimental_arch))]
-#![doc = include_str!("../README.md")]
+//! Supports the ESP32, ESP32-C2/C3/C6, ESP32-H2, ESP32-P4, and ESP32-S2/S3. Optional exception and
```

Comment:
Let's reword this a bit to introduce the crate properly, its a crate to get a back trace under panics and hardfaults. It supports a multitude of output transports depending on the various features chosen. That kind of thing. I'd also drop the supports X chip, that should be obvious from the chip features now.